components/backup-plugins/Gopkg.toml

```diff
@@ -26,9 +26,20 @@
 
 required = [
   "golang.org/x/tools/cmd/goimports",
-  "k8s.io/api"
 ]
 
+[[override]]
```

Comment:
Could you please group `constraint`s (direct dependencies) at the top of the file and add a comment “Transitive dependency of xyz...” before each override? That will make future dependencies updates so much easier.